apps/rush-lib/src/logic/TaskSelector.ts

```diff
@@ -114,15 +115,17 @@ export class TaskSelector {
     }
   }
 
-  private _registerAll(): void {
+  private _registerAll(addDependencies: boolean): void {
```

Comment:
is this condition inverted here? The option being passed in is ignore dependencies, but the param here is add dependencies

Reply:
something went wrong here. Will upload a followup.